src/Sleep.js

```diff
@@ -35,7 +33,7 @@ class Sleep {
       sumSoFar += dataItem.sleepQuality;
       return sumSoFar;
     }, 0)
-    return totalSleepQuality / sleepData.length
+    return totalSleepQuality / this.sleepData.length
```

Comment:
I might have already shifted this in my refactor, good catch though

Reply:
I don't think I would have caught that! Nice!